app/views/course/users/invitations.html.slim

```diff
@@ -6,7 +6,8 @@
 - accepted_course_users = @course_users.each.select { |user| !user.invited? }.size
 - progress = accepted_course_users * 100 / [@course_users.length, 1].max
 = display_progress_bar progress, [] do
-  = t('.progress', accepted: accepted_course_users, total: @course_users.size)
+  = t('course.users.invitations.progress', accepted: accepted_course_users,
```

Comment:
Since `display_progress_bar` renders a partial, the locale source was different - ie. `en.layouts.progress_bar.progress` if we use `.progress`. Discovered this when I was adding text to the course user badge as well. 

By default this should not be the case, but not sure if there is a way around this though? I could construct the entire `display_progress_bar` within the helper method itself >_<